src/extensions/surveys/surveys-utils.tsx

```diff
@@ -34,6 +34,16 @@ export function getSurveyResponseKey(questionId: string) {
     return `$survey_response_${questionId}`
 }
 
+function getSurveyIndexToIdMap(survey: Survey) {
+    return survey.questions.reduce<Record<number, string>>((acc, question, index) => {
```

Comment:
will the index matter here if the question is shuffled? just trying to avoid a bug If possible

Reply:
indexes ideally would match the $survey_questions as well

Reply:
Can we check if `$survey_questions` is being used in any insight, and if not, just fix that one?
eg
```
$survey_questions: [
	{
	id: UUID
	index: 1
	text: the text
	}
]
```

Reply:
removed this in favor of just using `survey_questions`